tests/e2e/simplekvtest.go

```diff
@@ -147,6 +156,27 @@ func kvGet(rtc client.RuntimeClient, key []byte) ([]byte, error) {
 	return resp.Value, nil
 }
 
+// kvInsertSpecialGreeting sends a transaction encoded in the keyvalue-special-greeting scheme.
+func kvInsertSpecialGreeting(rtc client.RuntimeClient, signer signature.Signer, greeting string) error {
+	sig, err := signer.ContextSign([]byte("keyvalue special greeting"), []byte(greeting))
+	if err != nil {
+		return fmt.Errorf("signing special greeting: %w", err)
+	}
+	utx := types.UnverifiedTransaction{
+		Body: cbor.Marshal(kvSpecialGreeting{
+			Greeting:  greeting,
+			From:      signer.Public().(ed25519.PublicKey),
+			Signature: sig,
+		}),
+		AuthProofs: nil,
```

Comment:
Shouldn't this configure the module auth proof?

Reply:
oh I must not have pushed that fix

Reply:
yeah. I fixed that earlier. must not have pushed it. (wrong account, reposting this)